<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="mainConsole">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="southButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QPushButton" name="westButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>41</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="northButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="teleportButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Teleport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eastButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>41</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewMap">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>View Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewCharInfo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>View Character Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewRoomItems">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>View Room Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewCharItems">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>View Character Items</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="staminaDisplay">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="selectSuspect1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>It's Suspect 1!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectSuspect2">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>300</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>It's Suspect 2!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multipleButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Demonstrate Multiple Inheritance(Check Terminal)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="constrExamplebutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Demonstrate Deep and Shallow Copy Constructors (Check Terminal)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="initListButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>61</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string>Demo
Initializer
List</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
